tests/rkt_api_service_test.go

```diff
@@ -230,7 +271,37 @@ func TestAPIServiceListInspectPods(t *testing.T) {
 		t.Errorf("Unexpected result: %v, should see zero pods", resp.Pods)
 	}
 
-	patchImportAndRun("rkt-inspect-print.aci", []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=42"}, t, ctx)
+	patches := []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=42"}
+	patchImportAndFetchHash("rkt-inspect-print.aci", patches, t, ctx)
+	iresp, err := c.ListImages(context.Background(), &v1alpha.ListImagesRequest{})
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+	if len(iresp.Images) == 0 {
+		t.Fatal("Expected non-zero images")
+	}
+	imgID, err := types.NewHash(iresp.Images[0].Id)
+	if err != nil {
+		t.Fatalf("Cannot generate types.Hash from %v: %v", iresp.Images[0].Id, err)
+	}
+	pm := schema.BlankPodManifest()
+	pm.Apps = schema.AppList{
+		schema.RuntimeApp{
+			Name: types.ACName("rkt-inspect"),
+			Image: schema.RuntimeImage{
+				Name: types.MustACIdentifier("coreos.com/rkt-inspect"),
+				ID:   *imgID,
+			},
+			Annotations: []types.Annotation{types.Annotation{Name: types.ACIdentifier("app-test"), Value: "app-test"}},
+		},
+	}
+	pm.Annotations = []types.Annotation{types.Annotation{Name: types.ACIdentifier("test"), Value: "test"}}
+	manifestFile := generatePodManifestFile(t, pm)
+	defer os.Remove(manifestFile)
+
+	runCmd := fmt.Sprintf("%s run --mds-register=false --pod-manifest=%s", ctx.Cmd(), manifestFile)
```

Comment:
@derekparker So since last couple of releases, `--mds-register=false` is the default, so we can delete that :)